# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/cookies/__dir__.ini
# Commit: 532731e94bb2
# Full Hash: 532731e94bb23b01b5e7136bee829844a0f08bc2
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2020-06-01 21:42:28
# Regressor Bug: 1638358
# File Overlap Count: 1
# Description:
#   Bug 1638358 - Cookie Schemeful Same-Site - part 5 - schemeful comparison, r=mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D75629
# ==============================================================================

diff -r fad2ba760157 -r 532731e94bb2 testing/web-platform/meta/cookies/__dir__.ini
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/meta/cookies/__dir__.ini	Mon Jun 01 11:43:28 2020 +0000
@@ -0,0 +1,1 @@
+prefs: [network.cookie.sameSite.schemeful:false]